### DISCUSSION TITLE
Linkerd HA mode causing service pods to be in OOMKill status

### DISCUSSION THREAD

**Discussion by shahriak** [linkerd/linkerd2#6231] [Q&A]
Hi,

I have a large cluster with 35 nodes for linkerd (100 ingress pods, 3 controller pods) and 38 workload nodes (32 service pods). During Helm installation, I am seeing the status of the service pods (with linkerd injected) go from Running -> OOMKilled -> CrashLoopBackOff and repeating...

After I remove the HA mode configs, I do not see this problem. I want to understand what is causing this and what is correct way to approach this? My assumption is that the HA mode has the cpu/memory limits defined and exceeding that causes the error.

Linkerd version: 2.10.0

**Comment by shahriak** [comment]
I upgraded to 2.10.2 version but issue still persists..

**Reply by olix0r** [reply]
HA mode will set memory limits that are not set otherwise, so this is likely why you are encountering OOM errors. It would be helpful to open an issue with as much detail about the workload that's encountering these problems as possible, especially including the output of `linkerd diagnostics proxy-metrics ...` on that pod before it crashes.
